main.go

```diff
@@ -1,14 +1,23 @@
 package main
 
 import (
+	"github.com/fatih/color"
 	"github.com/gojektech/proctor/cmd"
+	"github.com/gojektech/proctor/config"
 	"github.com/gojektech/proctor/daemon"
 	"github.com/gojektech/proctor/io"
 )
 
 func main() {
-	printer := io.NewPrinter()
-	proctorEngineClient := daemon.NewClient()
+	printer := io.GetPrinter()
+	proctorConfig, err := config.LoadConfig()
+	if (config.ConfigError{}) != err {
```

Comment:
The convention is to return an error when performing an operation and check if it isn't nil. We can remove the custom `ConfigError`, return the default error with relevant message and check if it isn't nil